sig-testing/flakiness-sla.md

```diff
@@ -0,0 +1,40 @@
+# Kubernetes Testing Flakiness SLA
+
+This document captures the expectations of the community about flakiness in
+our tests and our test infrastructure. It sets out an SLA (Service Level 
+Agreement) for flakiness in our tests, as well as actions that we will
+take when we are out of SLA.
+
+
+## Definition of a "Flake"
+
+We'll start by the definition of a _flake_. A test result is considered to
```

Comment:
I would want each environment we test to output some unique hash to know if it has actually changed.  For example, the version of kops can change rapidly over the life of a PR.